core/src/main/java/sem/group15/bubblebobble/core/BubbleBobble.java

```diff
@@ -57,8 +57,8 @@ public void render() {
 		controller.loop(Gdx.graphics.getDeltaTime(), batch);
         // draw score on screen
         bitmapFont.setColor(1, 1, 1, 1);
-		String scoreString = "score: " + controller.player.score;
-        bitmapFont.draw(batch, scoreString, 50, Gdx.graphics.getHeight() - 30);
+		String infoString = "score: " + controller.player.score + "    " + "Lives left: " + controller.player.lives;
```

Comment:
Does the job, maybe be consistent with capital letters- but we'll make a UI design review anyway so I'll leave it for now. <user-interface